include/image-commands.mk

```diff
@@ -383,6 +383,11 @@ define Build/kernel-bin
 	cp $< $@
 endef
 
+define Build/kernel-pack-npk
+	sh -c 'source $(STAGING_DIR_HOST)/share/npkpy-venv/bin/activate; python $(STAGING_DIR_HOST)/share/npkpy-venv/bin/pack_npk_kernel.py --kernel $@ --output $@.npk'
```

Comment:
I tried this locally and had an issue with `source`. I think it should be `.` (a dot).